src/main/java/org/opentripplanner/middleware/bugsnag/EventSummary.java

```diff
@@ -0,0 +1,45 @@
+package org.opentripplanner.middleware.bugsnag;
+
+import org.opentripplanner.middleware.bugsnag.response.EventException;
+import org.opentripplanner.middleware.models.BugsnagEvent;
+import org.opentripplanner.middleware.models.BugsnagProject;
+
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Event summary information provided to calling services. Response is based on static project and dynamic event
+ * information. Additional information relating to this can be found here:
+ * https://bugsnagapiv2.docs.apiary.io/#reference/organizations/collaborators/create-an-event-data-request
+ */
+public class EventSummary {
+
+    /** The error which this event is associated with */
+    public String errorId;
+
+    /** A list of {@link EventException) classes and messages associated with this event */
+    public List<EventException> exceptions;
+
+    /** The date/time Bugsnag received the event */
+    public Date received;
+
+    /** The project which this event is associated with */
+    public String projectId = "-1";
+
+    /** The associated project name */
+    public String projectName = "Unknown";
+
+    /** Associated environment e.g. Test, Dev, Production */
+    public String releaseStage;
+
+    public EventSummary(BugsnagProject project, BugsnagEvent bugsnagEvent) {
+        if (project != null) {
+            this.projectName = project.name;
+            this.projectId = project.id;
```

Comment:
I think another important project field is `htmlUrl`, so we can construct links in the UI that users can follow.

Reply:
Included.